### PR TITLE
Set feed_report for Cleverio pf100 to sum

### PR DESCRIPTION
**Pull request by FredrikM97**
## Summary

Solves https://github.com/home-assistant/core/issues/161566#issuecomment-4371447539

## Test plan

Locally

### CODE REVIEW

**Comment by FredrikM97** [comment]
@epenet do you have time to take a look at this? :)

File: src/tuya_device_handlers/devices/cwwsq/cwwsq_wfkzyy0evslzsmoi.py [hunk 1]
```diff
@@ -21,5 +22,16 @@
             )
         )
     )
+    .add_dpid_integer(
```
Comment (epenet) [inline src/tuya_device_handlers/devices/cwwsq/cwwsq_wfkzyy0evslzsmoi.py:25]: Can you add a test, highlighting that without the quirk the report type is none, and with the quirk it is sum?

Comment (FredrikM97) [reply src/tuya_device_handlers/devices/cwwsq/cwwsq_wfkzyy0evslzsmoi.py:25]: Sure!

**Comment by FredrikM97** [comment]
@epenet Added a test. Found an issue when local_strategy is None cause the test to crash.

**Review by epenet — APPROVED** [review]

**Comment by epenet** [comment]
I created release v0.0.26, but someone else now needs to do the corresponding bump in Home Assistant
